__tests__/feature/editing/propInfo.test.js

```diff
@@ -0,0 +1,42 @@
+import { renderApp, createHistory } from 'testUtils'
+import { screen } from '@testing-library/react'
+import * as sinopiaServer from 'sinopiaServer'
+import { getFixtureResourceTemplate } from 'fixtureLoaderHelper'
+
+jest.mock('sinopiaServer')
+// Mock jquery
+global.$ = jest.fn().mockReturnValue({ popover: jest.fn() })
+
+describe('getting property related info from a resorce', () => {
+  sinopiaServer.foundResourceTemplate.mockResolvedValue(true)
+  sinopiaServer.getResourceTemplate.mockImplementation(getFixtureResourceTemplate)
+
+  it('has tooltip text info or a link based on the content of a top-level property remark', async () => {
+    const history = createHistory(['/editor/resourceTemplate:testing:uber3'])
+    renderApp(null, history)
+
+    // // if the tooltip remark is text
```

Comment:
Any reason to have two forward slashes pairs (`//`) here?

Reply:
Just to indicate that it is a kind of nested test. I was under the impression that we still could not use ruby-styled describe- and it- blocks.